webview-src/index.ts

```diff
@@ -36,6 +36,33 @@ export async function error(message: string): Promise<void> {
   await log(LogLevel.Error, message)
 }
 
+interface RecordPayload {
+  level: number;
+  message: string;
+}
+
 export function attachConsole() {
```

Comment:
I don't really like the name of this function🤔

Reply:
maybe `bridgeToConsole`? also maybe check the log level and redirect to console.log, console.warn, console.error etc

Reply:
Is it not already checking that? I thought so 🤔

Reply:
oops :)